docs/it/schema.it.rst

```diff
@@ -13,8 +13,8 @@ Sezione ``conforme``
 Questa sezione contiene delle chiavi per auto dichiarare la conformità
 con la normativa vigente, rispetto ad alcune sezioni.
```

Comment:
Aggiungerei:

```
Se queste chiavi non vengono incluse, si intende che la conformità non è nota o non viene dichiarata.
```

Reply:
(Anche in inglese)

Reply:
In inglese lo schema.it.rst non è mai stato tradotto siccome contiene le chiavi italiane. Dici che è il caso di tradurlo? @alranel

Reply:
Fatto @alranel